packages/demo/package.json

```diff
@@ -21,6 +21,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@stencil-community/router": "^2.0.0-2"
+    "@stencil-community/router": "stencil-community/stencil-router"
```

Comment:
This doesn't actually work like I thought it would since this project is using workspaces. I'm having a hard time getting the demo running successfully since `2.0.0-2` hasn't been published on npm yet for the community package.

Any guidance on getting the router demo running would be appreciated